web/react/stores/user_store.jsx

```diff
@@ -164,13 +164,25 @@ var UserStore = assign({}, EventEmitter.prototype, {
     BrowserStore.setItem("sessions", sessions);
   },
   getSessions: function() {
-    return BrowserStore.getItem("sessions", []);
+    var sessions = BrowserStore.getItem("sessions");
+
+    if (sessions == null) {
+        return BrowserStore.getItem("sessions", {loading: true});
```

Comment:
The second parameter of BrowserStore.getItem is the default value so you only need this line. The same thing goes for getAudits below.